src/lib/conversion/rotation.h

```diff
@@ -117,7 +118,7 @@ const rot_lookup_t rot_lookup[] = {
 	{270,   0, 270 },
 	{180, 270,   0 },
 	{  0,  90, 180 },
-	{ 90,  90,   0 }
```

Comment:
yes I'm sure the removal is wrong ;) thanks. must have slipped during the merge conflict